### ISSUE TITLE
What is the ctc loss? The model is considered to be successfully trained

### ISSUE THREAD

**Issue by scuizhibin** [Alexander-H-Liu/End-to-end-ASR-Pytorch#75]

**Comment by vectominist** [comment]
You may refer to the paper [Connectionist Temporal Classification: Labelling Unsegmented Sequence Data with Recurrent Neural Networks](https://www.cs.toronto.edu/~graves/icml_2006.pdf) or [this website](https://distill.pub/2017/ctc/) to better understand CTC loss.
